frappe/recorder.py

```diff
@@ -151,7 +152,17 @@ def __init__(self):
 			self.method = frappe.request.method
 			self.headers = dict(frappe.local.request.headers)
 			self.form_dict = frappe.local.form_dict
+			self.event_type = "HTTP Request"
+		elif job := rq.get_current_job():
+			self.event_type = "Background Job"
+			kwargs = job._kwargs
```

Comment:
```suggestion
			kwargs = job.kwargs
```

Use public API wherever possible. They are same in this case?

Reply:
Ah I had copied from my sentry impl which I had taken from FC impl, will update both
(Probably will just create a separate util to get job name)

Reply:
https://github.com/ankush/frappe/blob/94f53d3f9149ad6f7e26d3b3c94ef62025a82452/frappe/utils/background_jobs.py#L207-L214

Add it here maybe, so it's usable everywhere. I haven't quite formalized what `frappe.job` is supposed to be. Right now it's just global for storing job related info and callback manager.